src/nemo-feedback/NemoFeedback.cc

```diff
@@ -508,11 +492,11 @@ void NemoFeedback::updateAltimeterSelection(std::vector<bool>& to_write) const {
   }
 }
 
-std::tuple<util::DateTime, size_t> NemoFeedback::mpiSync(size_t nLevelsLocal)
-  const {
+std::tuple<util::DateTime, size_t> NemoFeedback::mpiSync(
+    size_t nLevelsLocal) const {
   auto& comm = obsdb_.comm();
-  util::DateTime juldReferenceLocal = parameters_.refDate.value()
-    .value_or(util::DateTime{"1950-01-01T00:00:00Z"});
+  util::DateTime juldReferenceLocal = parameters_.refDate.value().value_or(
+      util::DateTime{"1950-01-01T00:00:00Z"});
   std::vector<util::DateTime> datetimes;
   obsdb_.get_db("MetaData", "dateTime", datetimes);
   juldReferenceLocal = parameters_.refDate.value().value_or(datetimes[0]);
```

Comment:
Line 502 seems to overwrite the result of 498/499. Can 498/499 be deleted and the util::DateTime part of 498 be moved to 502?